website/docs/audit.md

```diff
@@ -0,0 +1,69 @@
+---
+id: audit
+title: Audit
+---
+
+The audit functionality enables periodic evaluations of replicated resources against the policies enforced in the cluster to detect pre-existing misconfigurations. Audit results are stored as violations listed in the `status` field of the failed constraint.
+
+```yaml
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sRequiredLabels
+metadata:
+  name: ns-must-have-gk
+spec:
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Namespace"]
+  parameters:
+    labels: ["gatekeeper"]
+status:
+  auditTimestamp: "2019-05-11T01:46:13Z"
+  enforced: true
+  violations:
+  - enforcementAction: deny
+    kind: Namespace
+    message: 'you must provide labels: {"gatekeeper"}'
+    name: default
+  - enforcementAction: deny
+    kind: Namespace
+    message: 'you must provide labels: {"gatekeeper"}'
+    name: gatekeeper-system
+  - enforcementAction: deny
+    kind: Namespace
+    message: 'you must provide labels: {"gatekeeper"}'
+    name: kube-public
+  - enforcementAction: deny
+    kind: Namespace
+    message: 'you must provide labels: {"gatekeeper"}'
+    name: kube-system
+```
+
+- Audit interval: set `--audit-interval=123` (defaults to every `60` seconds)
+- Audit violations per constraint: set `--constraint-violations-limit=123` (defaults to `20`)
+- Disable: set `--audit-interval=0`
+
+By default, the audit will request each resource from the Kubernetes API during each cycle of the audit. To instead rely on the OPA cache, use the flag `--audit-from-cache=true`. Note that this requires replication of Kubernetes resources into OPA before they can be evaluated against the enforced policies. Refer to the [Replicating data](#replicating-data) section for more information.
+
+#### Audit using kinds specified in the constraints only
```

Comment:
Make this section heading more important (fewer #'s)?

Reply:
updated